src/ray/object_manager/plasma/object_lifecycle_manager.h

```diff
@@ -92,6 +93,9 @@ class IObjectLifecycleManager {
   ///
   /// \return true if object exists and reference count is greater than 0, false otherise.
   virtual bool RemoveReference(const ObjectID &object_id) = 0;
+
+  /// Returns true if object is spillable.
+  virtual bool IsObjectSpillable(const ObjectID &object_id) const = 0;
```

Comment:
Could you add more comments about `spillable`? An object which has been evicted and ref count is 1